common/src/app/common/types/container.cljc

```diff
@@ -15,13 +15,12 @@
 
 (s/def ::type #{:page :component})
 (s/def ::id uuid?)
-(s/def ::name string?)
-(s/def ::path (s/nilable string?))
+(s/def ::name ::us/string)
+(s/def ::path (s/nilable ::us/string))
 
 (s/def ::container
-  ;; (s/keys :req-un [::id ::name ::ctst/objects]
```

Comment:
I think this was commented out for performance, because validating spec of a tree of objects may be very slow.

And this validation is being triggered somewhere with real files in production.

I suppose it's good to activate it, but we need to be aware of this, and check when is this being triggered.